runtime/compiler/optimizer/VectorAPIExpansion.cpp

```diff
@@ -962,45 +1205,362 @@ TR_VectorAPIExpansion::validateVectorAliasClasses(TR_BitVector * vectorAliasTabl
+   bitsLength = vec_len_default;
+   objectType = Unknown;
+
+   int32_t refId = -1;
+
+   if (node->getOpCodeValue() == TR::aload ||
+       node->getOpCodeValue() == TR::astore)
+      {
+      refId = node->getSymbolReference()->getReferenceNumber();
+      int32_t classId = _aliasTable[refId]._classId;
+
+      if (classId <= 0)
+         return false;
+
+      if (_aliasTable[classId]._classId <= 0)
+         return false;
+
+      refId = _aliasTable[refId]._tempClassId;
+
+      if (refId <= 0)
+         return false;
+
+      if (_aliasTable[refId]._tempClassId <= 0)
+         return false;
+
+      }
+   else if (node->getOpCode().isFunctionCall() &&
+            isVectorAPIMethod(node->getSymbolReference()->getSymbol()->castToMethodSymbol()))
+      {
+      ncount_t nodeIndex = node->getGlobalIndex();
+      if (!_nodeTable[nodeIndex]._canVectorize)
+         return false;
+
+      refId = node->getSymbolReference()->getReferenceNumber();
+      int32_t classId = _aliasTable[refId]._classId;
+
+      if (classId <= 0)
+         return false;
+
+      if (_aliasTable[classId]._classId <= 0)
+         return false;
+
+      elementType = _nodeTable[nodeIndex]._elementType;
+      bitsLength = _nodeTable[nodeIndex]._vecLen;
+      objectType = _nodeTable[nodeIndex]._objectType;
+
+      return true;
+      }
+   else if (node->getOpCode().isVectorOpCode())
+      {
+      TR::SymbolReference *origSymRef = _nodeTable[node->getGlobalIndex()]._origSymRef;
+
+      if (!origSymRef)
+         return false;  // was vectorized earlier by auto-SIMD or another pass of VectorAPIExpansion
+
+      if (origSymRef->getSymbol()->isMethod())
+         {
+         ncount_t nodeIndex = node->getGlobalIndex();
+
+         elementType = _nodeTable[nodeIndex]._elementType;
+         bitsLength = _nodeTable[nodeIndex]._vecLen;
+         objectType = _nodeTable[nodeIndex]._objectType;
+         return true;
+         }
+
+      refId = origSymRef->getReferenceNumber();
+      }
+   else
+      {
+      return false;
+      }
+
+   elementType = _aliasTable[refId]._elementType;
+   bitsLength = _aliasTable[refId]._vecLen;
+   objectType = _aliasTable[refId]._objectType;
+
+   if (_trace)
+      traceMsg(comp(), "#%d bitsLength=%d\n", refId, bitsLength);
+
+   if (bitsLength != vec_len_unknown &&
+       bitsLength != vec_len_default &&
+       bitsLength != vec_len_boxed_unknown)
+      {
+      return true;
+      }
+
+   return false;
+   }
+
+void
+TR_VectorAPIExpansion::createClassesForBoxing(TR_ResolvedMethod *owningMethod, TR::DataType methodElementType,
+                                              vec_sz_t bitsLength)
+   {
+   if (methodElementType == TR::Int8 && bitsLength == 128 &&
+       _classByte128Vector == NULL)
+      {
+      _classByte128Vector = comp()->fej9()->getClassFromSignature("jdk/incubator/vector/Byte128Vector", 34, owningMethod, true);
+      TR_ASSERT_FATAL(_classByte128Vector, "Could not create Vector class from signature");
+
```

Comment:
Do you want an empty line here?
The same comment for the line after `TR_ASSERT_FATAL(_classByte128Mask, "Could not create Mask class from signature");`

Reply:
Fixed